soc/arm/nxp_imx/rt117x/Kconfig.series

```diff
@@ -0,0 +1,12 @@
+# iMX RT117x series
+
+# Copyright (c) 2020, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMX_RT117X
```

Comment:
I'd really like to see if we can do this without introducing a new SoC series. There is a lot of code duplicated from `soc/arm/nxp_imx/rt/`, so I would prefer if we can add RT117x there.